recipes/ffmpeg/all/conanfile.py

```diff
@@ -274,6 +275,8 @@ def build_configure(self):
                 args.append("--enable-libpulse" if self.options.pulse else "--disable-libpulse")
                 args.append("--enable-vaapi" if self.options.vaapi else "--disable-vaapi")
                 args.append("--enable-vdpau" if self.options.vdpau else "--disable-vdpau")
+
+            if self.settings.os == "Linux" or self.settings.os == "Macos":
```

Comment:
Simplify this, by removing this new line and changing the next one to:

`if self.options.get_safe("xcb"):`

Reply:
done in a force push 👍